_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -844,6 +844,7 @@ mfxStatus MFXVideoENCODE_EncodeFrameAsync(mfxSession session, mfxEncodeCtrl *ctr
     mfxStatus mfxRes;
 
     MFX_AUTO_LTRACE_WITHID(MFX_TRACE_LEVEL_API, "MFX_EncodeFrameAsync");
+    MFX_TRACE_CHROME_ADD_INFO("session", session);
```

Comment:
It seems you are trying to associate some name with variable, in the trivial case same name as variable name. Question: why? I believe you are trying to draw some dependency arrows in chrome.... So, do you look for variable names or variable values to draw them?

What's the reason why you can't use existing MFX_LTRACE_P/I/etc.? That's because there are too many of them or there is some other reason?